server/plugin/plugin.go

```diff
@@ -969,26 +968,34 @@ func (p *Plugin) HasUnreads(info *GitHubUserInfo) bool {
 func (p *Plugin) checkOrg(org string) error {
 	config := p.getConfiguration()
 
-	configOrg := strings.TrimSpace(config.GitHubOrg)
-	if configOrg != "" && configOrg != org && strings.ToLower(configOrg) != org {
-		return errors.Errorf("only repositories in the %v organization are supported", configOrg)
+	orgList := config.getOrganizations()
+	for _, configOrg := range orgList {
+		if configOrg == org {
+			return nil
+		}
 	}
 
-	return nil
+	return errors.Errorf("only repositories in the %v organization(s) are supported", config.GitHubOrg)
 }
 
 func (p *Plugin) isUserOrganizationMember(githubClient *github.Client, user *github.User, organization string) bool {
 	if organization == "" {
 		return false
 	}
 
-	isMember, _, err := githubClient.Organizations.IsMember(context.Background(), organization, *user.Login)
-	if err != nil {
-		p.client.Log.Warn("Failled to check if user is org member", "GitHub username", *user.Login, "error", err.Error())
-		return false
+	orgList := p.configuration.getOrganizations()
+	isMemberOfOrg := false
+	for _, org := range orgList {
+		isMember, _, err := githubClient.Organizations.IsMember(context.Background(), org, *user.Login)
```

Comment:
We've changed the meaning of this `isUserOrganizationMember` function here. Shouldn't we continue to just check if the user is a member of the `organization` parameter passed into the function?